notebooks/002-datamodules/002-btech.ipynb

```diff
@@ -0,0 +1,576 @@
+{
```

Comment:
<p><s>are</s> is</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>